mapper/src/main/java/jetbrains/jetpad/mapper/MappingContext.java

```diff
@@ -139,13 +138,8 @@ public void call(MappingContextListener l) {
     myProperties.put(property, value);
   }
 
-  @SuppressWarnings("unchecked")
   public <ValueT> ValueT get(MappingContextProperty<ValueT> property) {
-    Object value = myProperties.get(property);
-    if (value == null) {
-      throw new IllegalStateException("Property " + property + " not found");
-    }
-    return (ValueT)value;
+    return (ValueT) myProperties.get(property);
```

Comment:
My original intention was setting up these properties when context is created and treating them as assertion from mappers and synchronizers. Like this: 

    getMappingContext().get(HybridEditorsContext.PROPERTY)
      .createSynchronizer(this, source, target, spec)

It either succeeds or fails with a useful error message. I don't really want to wrap all `get`s with `if`s and NPEs without message are hard to trace in SuperDevMod.

Why do you need this change? I'm afraid that with introspection of `MappingContext` and properties we'll end up doing dependency management right in our mapping code.